src/tribler/core/utilities/main_thread_stack_tracking.py

```diff
@@ -0,0 +1,70 @@
+import linecache
+import sys
+from types import FrameType
+from typing import List, Optional, Tuple
+
+SWITCH_INTERVAL = 0.1
```

Comment:
NIT: Could you please consider adding a comment to this constant? It's not clear why the value is 0.1 and how it affects the logic of the module.

Reply:
I added the comment